### PR TITLE
Add OAuth Web Worker SDK

### PR DESCRIPTION
**Pull request by thivi**

### CODE REVIEW

File: oauth-web-worker/.eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+dist/
```
Comment (jeradrutnam) [inline oauth-web-worker/.eslintignore:1]: Missing new line at EOF

File: oauth-web-worker/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
+module.exports = {
```
Comment (jeradrutnam) [inline oauth-web-worker/.eslintrc.js:1]: Format file with 4 spaces indentation

File: oauth-web-worker/index.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
```
Comment (jeradrutnam) [inline oauth-web-worker/index.html:1]: Format file with 4 spaces indentation

File: oauth-web-worker/package.json [hunk 4]
```diff
@@ -0,0 +1,44 @@
+{
```
Comment (jeradrutnam) [inline oauth-web-worker/package.json:1]: Format file with 4 spaces indentation

File: oauth-web-worker/webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
+/**
```
Comment (jeradrutnam) [inline oauth-web-worker/webpack.config.js:1]: Format file with 4 spaces indentation

File: oauth-web-worker/tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (jeradrutnam) [inline oauth-web-worker/tsconfig.json:1]: Format file with 4 spaces indentation

File: oauth-web-worker/README.md [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+        console.log(error);
+    });
+```
+
+To make multiple api calls at once (this is similar to `axios.all` or `Promise.all`),
+
+```javascript
+const headers = {
+    Accept: "application/json",
+    "Access-Control-Allow-Origin": store.getState().config.deployment.clientHost
```
Comment (jeradrutnam) [inline oauth-web-worker/README.md:126]: This is only applicable for redux based projects and how the app is structured. Let's have a standard string here

**Comment by gayashanbc** [comment]
Closing this PR as considered obsolete

**Pull request closed without merging**